osidb/api_views.py

```diff
@@ -166,7 +168,13 @@ def get_valid_http_methods(cls: ViewSet, excluded: list[str] = None) -> list[str
         "or the CVE number corresponding to a Flaw"
     ),
 )
-
+jira_token_param = OpenApiParameter(
```

Comment:
This is a great idea, I think we should also add this `OpenApiParameter` for `Bugzilla-Api-Key` as currently it is kinda obfuscated what actually is needed for the operations that syncs to Bugzilla.

Reply:
I have changed and submitted in this PR in a separated commit, please take a look if it will work.